0.2.0/Dockerfile

```diff
@@ -0,0 +1,68 @@
+FROM openjdk:8-jdk
+
+ENV JANUS_VERSION=0.2.0 \
+    JANUS_HOME=/opt/janusgraph \
+    JANUS_CONFIG_DIR=/etc/opt/janusgraph \
+    JANUS_DATA_DIR=/var/lib/janusgraph \
+    JANUS_INDEXES="default" \
+    JANUS_INDEX_DEFAULT_BACKEND="lucene" \
```

Comment:
I think the big benefit of Lucene for a Docker image is that it can run embedded with JanusGraph. With Elasticsearch we would need a separate container. So, it wouldn't be possible to just do `docker run janusgraph` and get JanusGraph running with an index backend.